CHANGELOG.md

```diff
@@ -44,6 +44,33 @@ have stopped (or shortly will stop) testing with MSVC 2019 too.
   `PolicyBasedRoutingServiceClient`
 - [Retail](/google/cloud/retail/README.md) - new `AnalyticsServiceClient`
 
+### [Pub/Sub](/google/cloud/pubsub/README.md)
+
+- fix(pubsub): correctly trace modacks in the unary pull ([#14049](https://github.com/googleapis/google-cloud-cpp/pull/14049))
+
+### [Spanner](/google/cloud/spanner/README.md)
+
+- fix(spanner): avoid absl types in public Interval API ([#14095](https://github.com/googleapis/google-cloud-cpp/pull/14095))
+- feat(spanner): add a representation for the Spanner INTERVAL ([#14059](https://github.com/googleapis/google-cloud-cpp/pull/14059))
+- fix(spanner): delete idle sessions upon SessionPool destruction ([#14012](https://github.com/googleapis/google-cloud-cpp/pull/14012))
+
+### [Storage](/google/cloud/storage/README.md)
+
+- fix(storage): missing include ([#14052](https://github.com/googleapis/google-cloud-cpp/pull/14052))
+- fix(GCS+gRPC): MSVC build problems ([#14031](https://github.com/googleapis/google-cloud-cpp/pull/14031))
+- feat(GCS+gRPC): checksum options ([#14002](https://github.com/googleapis/google-cloud-cpp/pull/14002))
+- docs(GCS+gRPC): refactor AsyncClient comments ([#14001](https://github.com/googleapis/google-cloud-cpp/pull/14001))
+
+### [Common Libraries](/google/cloud/README.md)
+
+- feat(datastore): emulator environment variable ([#14100](https://github.com/googleapis/google-cloud-cpp/pull/14100))
```

Comment:
Consider a separate entry for the Datastore library.

Reply:
Done.  And did the same for "Storage Control", so there is no longer a "Common Libraries" section.